src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -304,6 +407,7 @@ def __init__(
         import weakref
 
         file_dict[self.file_pointer()] = weakref.ref(self)
+        self.parsed_description = parse_mvd(self.wrapped_data.header.file_description.description)
```

Comment:
We should not cache/store this here because the user can overwrite the description field. Just do the parsing as part of the mvd property getter.